### ISSUE TITLE
Xamarin Forms iOS Digits.Instance.Initialize is not working?

### ISSUE THREAD

**Issue by Awesometimes** [drungrin/Fabric.Sdk.Xamarin#34]
Hi,

I'm on the latest and greatest (PCL 4.6) and I've spent several hours trying to get iOS Digits to work, but I keep running into an exception on startup. It crashes when I invoke the line (from the AppDelegate's FinishedLaunching method):

&nbsp;&nbsp; Digits.Instance.Initialize(TwitterKey, TwitterSecret);

With the following exception:

&nbsp;&nbsp; Message: Object reference not set to an instance of an object
&nbsp;&nbsp; Source: DigitsKit.Platform
&nbsp;&nbsp; Stack Trace:  at DigitsKit.Initializer.Initialize (DigitsKit.IDigits digits, System.String consumerKey, System.String consumerSecret)

I was able to get the Android project to work with Crashlytics, Answers and Digits.  On iOS, I'm able to get Crashytlics and Answers to work, but just not Digits.

Any suggestions would be much appreciated and please let me know if you need anymore info.

**Comment by Awesometimes** [comment]
Just a minor update - the root of the problem seems to be that Bindings.DigitsKit.Digits.SharedInstance is always coming back as null, if that helps narrow down the troubleshooting.  I tried on both package versions 1.0 and 1.1 and get the same results.

When I explicitly try to create a new instance of Digits, I get the error:

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; System.Exception: Could not create an native instance of the type 'Bindings.DigitsKit.Digits': the native class hasn't been loaded.

I looked up the that error message and it seems to have to do with architecture.  All this time, I'd been running it on my simulator and not my physical device.  I tried it on my phone and it loaded up fine.

I'd like to note, after running it, I ran into an error when calling Digits.Instance.Authenticate, with the following error: 

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Terminating app due to uncaught exception 'NSUnknownKeyException', reason: '[ setValue:forUndefinedKey:]: this class is not key value coding-compliant for the key countryButton.'

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  Reference:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  https://twittercommunity.com/t/dgtauthviewcontroller-0x14ed00b0-setvalue-forundefinedkey-problem/54830

Seems to have to do with an outdated DigitsKitResources.bundle. I downgraded to 1.0 on the iOS project to workaround this error.

FWIW, I tried the https://github.com/joeisapro/MonoTouch.Fabric Digits implementation and that worked on both the simulator and device.
